scrapy/utils/iterators.py

```diff
@@ -48,7 +48,7 @@ def xmliter_lxml(obj, nodename, namespace=None, prefix='x'):
     iterable = etree.iterparse(reader, tag=tag, encoding=reader.encoding)
     selxpath = '//' + ('%s:%s' % (prefix, nodename) if namespace else nodename)
     for _, node in iterable:
-        nodetext = etree.tostring(node)
+        nodetext = etree.tostring(node, encoding=six.text_type)
```

Comment:
Hm, this doesn't look right -- `six.text_type` is `unicode` in PY2 and `str` in PY3.

Perhaps this should be the reader.encoding used in iterparse above?